data/hai missions.txt

```diff
@@ -1061,3 +1061,252 @@ mission "Hai Honeymoon"
+			`	While you thought she would've picked one or two jobs quickly and headed back to her ship, she keeps going through them and double checking the information for each one, as if something about them bothers her.`
+			`	"What's wrong? Are you struggling to pick between two different jobs?" you ask.`
+			`	"These all seem very easy and I could even accept many at a time, but the problem is their pay isn't very good." She grabs some worn papers from her pocket, and shows it to you: they're the documents for her loan. "The bank back home would only give me enough for the ship if I agreed on a short-term loan. I lost two days before you came along to help me, and if I don't get some better paying job soon, I won't even be able to pay the interest."`
+			`	As you're trying to figure out a way to get the most out of the available missions, you feel a tap on your shoulder, and turn to see a Hai standing beside a human couple. The couple introduce themselves as Xavier and Vanessa, who are here with their Hai friend Narancia. "We overheard your conversation and are willing to pay a little extra for a job. We're part of a group organizing a big festival in Allhome to celebrate the friendship between humans and Hai, and we just got a last-minute idea: we've already arranged for lots of human cuisine to be transported there, and we'd like to get our hands on some spices the Quarg have in the ringworld here."`
+			`	They tell Nanachi that they won't be going along, they already have a transport to Allhome, and will stay here a bit longer to try and advertise the event.`
+				accept
+	on stopover
+		dialog
+			`A spaceport worker contacted Nanachi shortly before you two landed and made sure the spices were ready to be loaded in her ship. You help the workers and Nanachi make sure everything is in order, and prepare for your trip to Allhome.`
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
+	on complete
+		conversation
+			`Allhome's docking area shows much more traffic than usual, many captains here are helping workers open cargo containers full of food, clothes, books, movies and even some mounted animals from human space. You finally meet up with the festival's organizers, who seem very excited to have acquired the exotic condiments, and have invited you two to a bar in the spaceport for a drink.`
```

Comment:
```suggestion
			`Allhome's docking area shows much more traffic than usual. Many captains here are helping workers open cargo containers full of food, clothes, books, movies and even some mounted animals from human space. You finally meet up with the festival's organizers, who seem very excited to have acquired the exotic condiments, and have invited you two to a bar in the spaceport for a drink.`
```
I feel like instead of having the player have to break the narrative and click the spaceport button, this mission could just be without an `on complete` conversation, and the next mission could be marked as `landing` with the conversation here being the start of the next mission.

Reply:
The suggested change is done (`, m` to `. M`)

Personally, I'm not a fan of going from deadline mission to deadline mission using `landing`.  If something bad happened in the prior deadline mission, you'd want some time to go and try to correct it.  As such, I'm going to leave that as-is.

Reply:
Not a fan of going directly from Deadline to Deadline missions without some chance of a break in-between them.  I'm going to leave as-is.

`, m` to `. M` was changed.